src/components/SpacerView.tsx

```diff
@@ -40,10 +33,8 @@ function SpacerView({shouldShow = true, style = []}) {
         borderBottomWidth.value = withTiming(values.borderBottomWidth, {duration});
     }, [shouldShow, borderBottomWidth, marginVertical]);
 
-    return <Animated.View style={[animatedStyles, ...StyleUtils.parseStyleAsArray(style)]} />;
+    return <Animated.View style={[animatedStyles, style]} />;
```

Comment:
I am not sure that `View` handles `[ [ { } ] ]` sub-arrays.
```suggestion
    return <Animated.View style={[animatedStyles, ...style]} />;
```